src/Views/BasicNumbersInput.js

```diff
@@ -0,0 +1,50 @@
+import { Console } from '@woowacourse/mission-utils';
+import throwError from '../Utils/throwError.js';
+import Rules from '../Utils/Rules.js';
+import Errors from '../Constants/Errors.js';
+import { InputComment } from '../Constants/display.js';
+
+const BasicNumbersInput = {
+  get: async () => {
+    const userInput = await Console.readLineAsync(InputComment.BASIC_NUMBERS);
+    return userInput;
+  },
+
+  validate: (basicNumbersInput) => {
+    if (Rules.isNoValueString(basicNumbersInput))
+      throwError(Errors.BasicNumbers.NO_INPUT);
+
+    const basicNumbers = basicNumbersInput.split(',');
```

Comment:
```suggestion
    const basicNumbers = basicNumbersInput.split(InputComment.Seperator.VALUE);
```
상수화 하셨던걸 사용하는건 어떨까요?!

Reply:
오잉?! 완전 놓쳤습니다 ㅜㅜ